examples/ticks_and_spines/major_minor_demo.py

```diff
@@ -1,84 +1,73 @@
-"""
+r"""
 ================
-Major Minor Demo
+Major/Minor Demo
```

Comment:
```suggestion
Major and minor ticks
```

I'm in favor of using more telling names than "XY Demo". "Demo" does not add any semantic information. All examples are some sort of demo.

Reply:
agreed, edited